docs/contracts.rst

```diff
@@ -694,8 +694,9 @@ Taking the following contract code as an example:
 
     >>> array_contract.functions.getBytes2Value().call()
     [b'b\x00']
-    >>> array_contract.functions.setBytes2Value([b'a']).transact()
-    HexBytes('0x977bd66e0ef59433a02c2f71f61ec3ca48bf7a5a19d10aece84570061fe9d4f3')
+    >>> tx_hash = array_contract.functions.setBytes2Value([b'a']).transact({'gasPrice': 200000})
+    >>> tx_dict = w3.eth.getTransaction(tx_hash)
+    >>> assert tx_hash == tx_dict.hash
```

Comment:
This assertion looks a little superfluous, it just checks if `getTransaction()` isn't broken I guess?

Reply:
Oops, you're right. This was a remnant from debugging. Thanks!